css-two-column-layout/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1>SCALE, SPEED, AGILITY</h1>
+        <p class="header">We've dedicated thousands of man hours to making healthcare
+        institutions in the Helium network run 10x more efficiently and deliver
+        an unparalleled quality of care.</p>
+      </div>
+    </div>
+    <div class="row">
+      <img src="images/software.svg" alt="software">
+      <div class="article">
+        <div class="column-half">
+          <h3>INTUITIVE, SMART SOFTWARE</h3>
+          <p>We build products that transform how healthcare facilities run across
+            Africa - making them radically more efficient operationally and
+            financially, a complete turnaround.</p>
+        </div>
+      </div>
```

Comment:
make sure that you have `column` elements wrapping the content. Dont forget your boxes!